### ISSUE TITLE
max option no longer works in 0.2.0

### ISSUE THREAD

**Issue by fzammetti** [nextapps-de/winbox#64]
Hi,

It appears there is a breakage in 0.2.0 from 0.1.9: setting max:true on initial window creation results in nothing being rendered. 
 When max is false, the window renders fine.  But, when true, nothing appears.  

I've attached an example that includes the 0.2.0 bundle.  Copy in the 0.1.9 bundle and you'll find that it works as expected.

[max_test.zip](https://github.com/nextapps-de/winbox/files/7190961/max_test.zip)

Tested in the latest version of Edge and Chrome as of 9/19/2001.

I did a quick compare of the source between 0.1.9 and 0.2.0 and there's really not too many changes, looks like stuff mostly related to the split screen feature, but I didn't find any obvious problems.  I tried a few things in the maximize function, since I'm guessing that's where the problem likely is, but I couldn't get it to work.

Unfortunately, this completely breaks an app I'm developing, so I'll have to revert to 0.1.9 in the meantime, but it would be the only dependency not on its most current version then, so I'm hoping a fix is quick and easy :)

Thanks,
Frank

**Comment by kodie** [comment]
@fzammetti I just tried out your example. The reason why you're having issues is because there's no other content on the page, and since the WinBox element gets its position set to `absolute`, the page height is essentially 0. In your example you can fix this by adding `style="width:100vw;height:100vh;"` to the `body` element.

**Comment by fzammetti** [comment]
So, it's weird... I upgraded to 0.2.1 in my little sample app and I DID NOT needed to add that style, it just worked as expected.  But, in my actual project where I first discovered this, I DID need to add it (also had to set padding and margin to 0 on the body to avoid scrollbars when the window was maxed, but that's fine).  I'm not sure what the difference is, but I'll take it :)  Thanks, I'm good to go now, closing.
